demos/src/demo-donut.js

```diff
@@ -142,8 +171,15 @@ function createDonutWithHighlightSliceChart() {
             .width(containerWidth)
             .height(containerWidth/1.8)
             .externalRadius(containerWidth/5)
-            .internalRadius(containerWidth/10);
+            .internalRadius(containerWidth/10)
+            .on('customMouseOver', function (data) {
```

Comment:
It seems a bit weird with `data.data.id`, maybe renaming the callback argument as `response` or `payload` would help.

Reply:
Good call, changing it